allure-report-face/src/components/status-toggle/StatusToggleView.js

```diff
@@ -15,7 +15,7 @@ class StatusToggleView extends ItemView {
     serializeData() {
         const statuses = settings.get('visibleStatuses');
         return {
-            statistics: Object.keys(this.statistics).map( status => ({
+            statistics: Object.keys(this.statistics).filter(status => status !== 'total').map( status => ({
```

Comment:
its better to use array with statuses instead of `Object.keys(this.statistics)`, to specify right statuses order